src/components/ServicePane.js

```diff
@@ -58,9 +60,16 @@ export default class ServicePane extends React.Component {
 
                     <CollapsedOpeningTimes object={object.open} />
                     <hr />
-                    <Address {...object.location} />
+                    <Address location={object.Location()} />
                     <hr />
                     <ContactMethods object={object} />
+                    <hr />
+                    <GoogleMapsLink
```

Comment:
Note in mine I was linking to the Google Maps directions (starting point "Current Location")